package.json

```diff
@@ -664,6 +716,29 @@
                     "type": "boolean",
                     "default": false,
                     "description": "Enables the Advanced Web App Creation which will prompt for app name, location, website OS, runtime, resource group, and app service plan."
+                },
+                "appService.connections": {
+                    "type": "array",
+                    "description": "Array of web apps with list of all theirs connections",
+                    "items": {
+                        "type": "object",
+                        "description": "All connections of specific web app.",
+                        "additionalProperties": false,
```

Comment:
Make sure 'webAppId' is required. I think you just need to set `"required": ["webAppId"]`